t/pg.t

```diff
@@ -405,7 +405,7 @@ subtest 'Stats' => sub {
   is $minion->stats->{inactive_workers}, 1, 'one inactive worker';
   $minion->enqueue('fail');
   is $minion->stats->{enqueued_jobs}, 1, 'one enqueued job';
-  $minion->enqueue('fail');
+  $minion->enqueue('fail' => [] => {queue => 'queue1'});
```

Comment:
Is there a good reason to change existing tests?

Reply:
These are not standalone tests, but rather a scenario. 
And if you follow logic of the modified scenario - it must be an improvement that it now also covers additional queues besides 'default'. And it is hard to add checks for new functionality without affecting whole scenario. And it is not that beneficial to have bunch of similar scenarios which are slightly different. So, it is hard to guess what exactly you would accept without questions - so feel free to adjust it to the acceptable criteria.